### PR TITLE
Fixes #31099 - filter out unstable facts: memory and load

### PR DESCRIPTION
**Pull request by lzap**
Filter out puppet facts which are unstable. For more info read: https://community.theforeman.org/t/facter-4-and-unstable-facts/20900

### CODE REVIEW

File: app/models/setting/provisioning.rb [hunk 1]
```diff
@@ -110,7 +121,7 @@ def self.map_pxe_kind
     end
   end
 
-  def self.default_excluded_facts(ignored_interfaces = IGNORED_INTERFACES)
-    ignored_interfaces
+  def self.default_excluded_facts
```
Comment (ezr-ondrej) [inline app/models/setting/provisioning.rb:124]: do we really want to drop the possibility to override the default?

Comment (lzap) [reply app/models/setting/provisioning.rb:124]: I searched the codebase and I could not find use of this, prove me wrong.

Comment (tbrisker) [reply app/models/setting/provisioning.rb:124]: I'm fine with dropping it, as far as i can tell this is only used for the default value for `excluded_facts` setting.

Comment (lzap) [reply app/models/setting/provisioning.rb:124]: Yeah that's what I thought. We added this but never used, to me it's rather confusing so I removed it.

**Comment by lzap** [comment]
Tests are green btw.

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !

**Comment by tbrisker** [comment]
release note please

**Review by ezr-ondrej — APPROVED** [review]
LGTM :+1: Thanks @lzap!

**Pull request merged**